src/main.zig

```diff
@@ -1,7 +1,118 @@
 const std = @import("std");
+const sdl = @import("sdl2");
+const chocl8 = @import("chocl8.zig");
+
+fn draw(renderer: sdl.Renderer, fbuf: [32]u64) anyerror!void {
+    var k: u7 = 0;
+    try renderer.setColor(sdl.Color.white);
+    while (k < 64) : (k += 1) {
+        var j: u7 = 0;
+        while (j < fbuf.len) : (j += 1) {
+            if ((fbuf[j] & (@as(u64, 1) << @truncate(u6, 64 - k))) != 0) {
+                try renderer.fillRect(sdl.Rectangle{
+                    .x = @as(c_int, k) * 10,
+                    .y = @as(c_int, j) * 10,
+                    .width = 10,
+                    .height = 10,
+                });
+            }
+        }
+    }
+}
 
 pub fn main() anyerror!void {
-    std.log.info("All your codebase are belong to us.", .{});
+    try sdl.init(.{
+        .video = true,
+        .events = true,
+        .audio = true,
+    });
+    defer sdl.quit();
+
+    const window = try sdl.createWindow(
+        "chocl8",
+        .{ .centered = {} },
+        .{ .centered = {} },
+        640,
+        320,
+        .{ .shown = true },
+    );
+    defer window.destroy();
+
+    const renderer = try sdl.createRenderer(window, null, .{ .accelerated = true });
+    defer renderer.destroy();
+
+    if (std.os.argv.len > 1) {
```

Comment:
It would be important to note that `std.os.argv` doesn't work for Windows, or any platform that doesn't have a posix/linux equivalent of "argv". If you want proper cross-platform code, consider utilizing `std.process.args`, or `std.process.argsAlloc` paired with `std.process.argsFree`.